.github/workflows/main.yaml

```diff
@@ -51,6 +51,7 @@ jobs:
       - name: Build and publish Docker image
         uses: nais/docker-build-push@v0
         id: docker-build-push
+        if: (!startsWith(github.ref_name, 'dependabot'))
         with:
           team: teamsykefravr
           identity_provider: ${{ secrets.NAIS_WORKLOAD_IDENTITY_PROVIDER }}
```

Comment:
Dette steget feiler for Dependabot pull requests fordi de ikke har denne secreten.

Reply:
OK, men det vll virke når en av oss merger PR'en?

Reply:
Vil tro det, men vi får nesten bare teste 😬